setup.py

```diff
@@ -83,6 +83,10 @@ class BuildExt(build_ext):
         "unix": ["-O3", "-W", "-fPIC", "-shared", "-fopenmp"],
     }
 
+    if os.environ.get("ENABLE_WARNINGS", False):
+        c_opts["unix"].extend(["-Wall", "-Wextra", "-Werror"])
+        l_opts["unix"].extend(["-Wall", "-Wextra", "-Werror"])
+
```

Comment:
```suggestion
        c_opts["msvc"].extend(["/Wall", "/WX"])
        l_opts["msvc"].extend(["/WX"])
```
See [here](https://docs.microsoft.com/en-us/cpp/build/reference/compiler-option-warning-level?view=msvc-160) for MSVC flags. I assume the `/` is correctly parsed and does not need to be escaped, but if failures occur it may be worth a try.